src/storage/storageController.ts

```diff
@@ -32,6 +32,15 @@ export type ConnectionsFromStorage = {
   [connectionId: string]: StoreConnectionInfo;
 };
 
+export const SecretStorageLocation = {
+  Keytar: 'vscode.Keytar',
+  SecretStorage: 'vscode.SecretStorage',
+} as const;
+
+export type SecretStorageLocationType =
+  | typeof SecretStorageLocation.Keytar
+  | typeof SecretStorageLocation.SecretStorage;
+
```

Comment:
Could it be just `export type SecretStorageLocationType = 'vscode.Keytar' | 'vscode.SecretStorage';`? Do we need at all SecretStorageLocation? cc @addaleax

Reply:
Yea I think it could be. I probably did it this way just so I also have a string const to use in places where the types are not available from the context (for example tests and a couple other places in the code as well where we declare a connection info like object with secret storage).